packages/reporter/src/collapsible/collapsible.tsx

```diff
@@ -5,9 +5,11 @@ import { onEnterOrSpace } from '../lib/util'
 
 interface Props {
   isOpen?: boolean
+  headerWrapperClass?: string
```

Comment:
Instead of adding a separate `headerWrapperClass` prop, let's only use the `headerClass` and move it to the wrapper. Then any usage that needs to target the 'header' in css can use `.<headerClass> .collapsible-header`.

Reply:
@chrisbreiding I tried to accomplish something like this, but I was struggling because I need to apply different styles to the wrapper and where the existing header is. The only style needed for the wrapper is `display: flex`, but when this is applied it causes some slight display differences with other collapsible. Alternatively, I think the best option might be to apply `display: flex` to the wrapper no matter what, and easily update the styles for other collapsibles so that they play nicely with this wrapper.